### PR TITLE
US618048: Updated copyright to OpenText and 2023

### PR DESCRIPTION
**Pull request by michael-bryson**
Ticket: https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=618048

### CODE REVIEW

**Comment by michael-bryson** [comment]
FYI, this is being used here: https://github.com/CAFapi/opensuse-base-image/pull/50

File: pom.xml [hunk 1]
```diff
@@ -32,11 +32,6 @@
 
     <inceptionYear>2018</inceptionYear>
 
-    <organization>
-        <name>EntIT Software LLC, a Micro Focus company</name>
-        <url>https://www.microfocus.com</url>
-    </organization>
```
Comment (dermot-hardy) [inline pom.xml:38]: Why is this being removed instead of updated?

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (michael-bryson) [reply pom.xml:38]: I wasn't sure about this, it looks like it hasn't been updated in a long time so must not be in use as EntIT has not been used for a long time. If its safer I can update this, is this ok?
```
    <organization>
        <name>Open Text</name>
        <url>https://www.opentext.com</url>
    </organization>
```

Comment (dermot-hardy) [reply pom.xml:38]: It can't be removed.  It is a Maven Central requirement so removing it would cause auto-release to fail.

File: pom.xml [hunk 2]
```diff
@@ -33,8 +33,8 @@
     <inceptionYear>2018</inceptionYear>
 
     <organization>
-        <name>EntIT Software LLC, a Micro Focus company</name>
-        <url>https://www.microfocus.com</url>
+        <name>Open Text</name>
```
Comment (dermot-hardy) [inline pom.xml]: Change this to "Open Text Corporation"

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**